src/index.html

```diff
@@ -11,7 +11,104 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="fieldset">
+        <legend class="leg">Personal information</legend>
+        <label class="form">
+          Surname:
+          <input
+           type="text"
+           name="surname"
+           autocomplete="off"
+          >
+        </label>
+        <label class="form">
+       Name:
+          <input type="text" name="name"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form">
+       How old are You?
+          <input type="number" name="age"
+                 value="12"
+                 min="1" max="100"
+          >
+        </label>
+        <label class="form">
+        Full date of birth:
+          <input type="date" name="date"
+            placeholder="mm/dd/yyyy"
+          >
+        </label>
+        <label class="form">
+        I accept the term of the agreement
+          <input type="checkbox" name="accept"
+              required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form">
+        E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          >
+        </label>
+        <label class="form">
+        Password:
+          <input type="password"  name="password" >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="form">
```

Comment:
This is redundant label. You have used it for answers
```suggestion
        <div class="form">
```